swift-tests/SnapshotTests/LottieSnapshotTests.swift

```diff
@@ -205,3 +221,22 @@ extension UIImage {
         self.init(cgImage: cgImage, scale: 1.0, orientation: .up)
     }
 }
+
+extension Colorspace {
+    /* Note: Despite the unconventional bitmap configuration settings, the resulting snapshots produce correctly rendered Lottie frames. 
+
+     Typically, ARGB should use a big endian format since the red value follows the alpha at the smallest memory address.
+     Conversely, ABGR should utilize a little endian format with `premultipliedFirst` due to blue being the first color channel following alpha.
+
+     However, results from the snapshots indicate correct visual output, suggesting a potential anomaly in iOS’s handling of memory format.
+     Verification confirms that ThorVG is outputting buffers in the correct ARGB/ABGR format, pointing to an iOS-specific issue in interpreting these formats.
+     */
+    var bitmapInfo: CGBitmapInfo {
+        switch self {
+        case .argb:
+            return [.byteOrder32Little, CGBitmapInfo(rawValue: CGImageAlphaInfo.premultipliedFirst.rawValue)]
+        case .abgr:
+            return [.byteOrder32Big, CGBitmapInfo(rawValue: CGImageAlphaInfo.premultipliedLast.rawValue)]
+        }
+    }
+}
```

Comment:
Just a note here: we've found a weird bug with iOS in aligning the pixel format when trying to create a `CGBitmapInfo` for our `CGContext` when taking screenshots. I will continue to investigate this further and create a bug report with Apple if neccessary.

Reply:
The Core Graphics logic seems fine to me. For our use case, you can think of the CGBitmapInfo as a way to interpret the color information stored in the buffer. Since we are using RGB,  here’s how the different values of `CGImageAlphaInfo` will behave for `.byteOrder32Little`:

```
none: RGB
first: ARGB
last: RGBA
alphaOnly: A
noneSkipFirst: _RGB // _ means to ignore whatever value is in there
noneSkipLast: RGB_
``` 

For `.byteOrder32Big`, the result will be the same values as before, but reversed. 

So, to summarize, if you want to create a `CGBitmapInfo` for a particular pixel format (other than only A), you can follow these steps: 
1. Are the RGB components in that order (R first and B last) in the pixel format? If yes, use `.byteOrder32Little`, otherwise use `.byteOrder32Big`.
2. If the RGB components are not in that same order in the pixel format, reverse the pixel format. 
3. Is the alpha at the start of the pixel format obtained in step 2? If yes, use `.premultipliedFirst`, otherwise use `.premultipliedLast`.

Reply:
Thanks for such a detailed comment @rlaguilar. I really appreciate how you take the time to walk us through how these conventions work. 

I think the core thing I was missing was that iOS follows the little-endian format. Which means that reading a buffer of color values would require us to read it in `.byteOrder32Little` to get the RGB order. Which now makes more sense to me.

And I love the steps! Will save those as a guide that I will definitely need to come back to in the future. 😅 

Have removed the comment 👍